README.md

```diff
@@ -11,40 +64,41 @@ The path follows the pattern: ``clients/{language}/github.com/{owner}/{repositor
 The ``{language}`` component of the path is the path-safe representation
 of the full language name which is mapped in [languages.json](./languages.json).
 
-Each client's JSON object represents the details displayed on the [clients documentation page](https://valkey.io/docs/clients).
+Each client's JSON object represents the details displayed on the [clients documentation page](https://valkey.io/clients/).
 
-For example [clients/python/github.com/placeholderkv](./clients/python/github.com/placeholderkv/placeholderkv-py.json):
+For example [clients/python/github.com/valkey-io/valkey-go.json](./clients/python/github.com/valkey-io/valkey-go.json):
 
-```
+```json
 {
-    "name": "placeholderkv-py",
-    "description": "Mature and supported. Currently the way to go for Python.",
+    "name": "valkey-go",
+    "description": "A fast Golang Valkey client that supports Client Side Caching and Auto Pipelining.",
     "recommended": true
 }
 ```
 
-## Commands
+Modules, libraries and tools follow a similar structure under their respective directories.
+
+### Commands
+
+The command pages under `commands/` in this repo are not complete without some
+metadata from the Valkey repo, namely the JSON files in the `src/commands/`
+folder. The content of these JSON files is combined with the Markdown files in
+this repo when the documentation is rendered.
 
-Valkey commands are described in the `commands.json` file that is auto generated
-from the Valkey repo based on the JSON files in the commands folder.
-See: https://github.com/valkey-io/valkey/tree/unstable/src/commands
-See: https://github.com/valkey-io/valkey/tree/unstable/utils/generate-commands-json.py
+See: https://github.com/valkey-io/valkey/tree/unstable/src/commands/
 
 For each command there's a Markdown file with a complete, human-readable
 description.
-We process this Markdown to provide a better experience, so some things to take
-into account:
+We process this files to provide a better experience.
 
 *   Inside text, all commands should be written in all caps, in between
     backticks.
     For example: `INCR`.
 
-*   You can use some magic keywords to name common elements in Valkey.
-    For example: `@multi-bulk-reply`.
-    These keywords will get expanded and auto-linked to relevant parts of the
-    documentation.
-
+The reply types and descriptsions are stored in separate JSON files in this doc repo.
```

Comment:
```suggestion
The reply types and descriptions are stored in separate JSON files in this doc repo.
```

Reply:
So many typos! It's my specialty. :upside_down_face: